app/client/styles/components/_Navigation.scss

```diff
@@ -6,7 +6,7 @@
     display: flex;
     align-items: center;
     padding: 8px 50px;
-    background-color: #191b1c;
+    background-color: $background-color-dark;
```

Comment:
Oh, good to know that these variables are all global. I didn't realize that before.